src/cli/install.rs

```diff
@@ -25,6 +25,10 @@ pub struct Args {
 
     #[arg(long, short, conflicts_with = "environment")]
     pub all: bool,
+
+    /// Do not install local path dependencies. Requires --frozen
+    #[arg(long, requires = "frozen")]
```

Comment:
Why would `--locked` not be correct here?

Reply:
Because with `--locked`, pixi will need the path dependency when checking the lockfile. which does not work if said dependency is not present.